nre/bpf/nodefw/nodefw/src/main.rs

```diff
@@ -15,12 +15,16 @@ use std::sync::{Arc, RwLock};
 use tokio_util::sync::CancellationToken;
 use tracing::{debug, info, warn};
 
+pub const KILLSWITCH_FILENAME: &str = "__FW_KILLSWITCH__";
+
 #[derive(Debug, Parser)]
 struct Opt {
     #[clap(short, long, default_value = "lo")]
     iface: String,
     #[clap(short, long, value_enum, default_value_t=Mode::Default)]
     mode: Mode,
+    #[clap(short, long)]
+    killswitch_path: String,
```

Comment:
can we rename this to drain_file.  killswitch implies a non-graceful shutdown when i read it.